Solutions/Apache Log4j Vulnerability Detection/Hunting Queries/WAF_log4j_vulnerability.yaml

```diff
@@ -1,6 +1,8 @@
 id: 1d4d383e-0ca6-4d3a-a861-8f37aeef18cb
 name: Azure WAF Log4j CVE-2021-44228 hunting
 description: |
+   'This hunting query looks in Azure Web Application Firewall data to find possible exploitation attempts for CVE-2021-44228 involving log4j vulnerability.'
```

Comment:
"This hunting query searches possible exploitation attempts for CVE-2021-44228 involving log4j vulnerability in Azure Web Application Firewall logs."

Reply:
Resolved